### PR TITLE
beautified member section

### PR DESCRIPTION
**Pull request by akshat0047**
Issue #33 

The member section of the website is improved by implementing:
- card structure
- gradient background to cards
- member image, bio and links in grid manner
- responsiveness

Please review for changes.
Demo- https://akshat0047.github.io/moz-hack-3/

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -424,16 +424,76 @@ <h1 class="title is-1"> Past Sponsors</h1>
...
+            <div class="col-4 col-sm-4 col-md-4 col-lg-4">
+              <img src="img/team/Suhail.jpg" class="res-img">
+            </div>
+            <div class="col-8 col-sm-8 col-md-8 col-lg-8 ">
+              <h6 class="member-name">NAME</h6>
+              <hr class="card-divider"/>
+              <p class="member-bio">BIO GOES HERE</p>
+              <span class="icon-box">
+                  <a href="#"><i class="fab fa-facebook icon "></i></a>
+                  <a href="#"><i class="fab fa-instagram icon "></i></a>
```
Comment (championshuttler) [inline index.html]: I think we will not need instagram profile here. What's your say @suhailsinghbains ? ;)

File: index.html [hunk 2]
```diff
@@ -424,16 +424,76 @@ <h1 class="title is-1"> Past Sponsors</h1>
...
+                  <a href="#"><i class="fab fa-github-square icon "></i></a>
+                  <a href="#"><i class="fab fa-pinterest icon "></i></a>
```
Comment (championshuttler) [inline index.html]: Pinterest as well :)

File: index.html [hunk 3]
```diff
@@ -424,16 +424,76 @@ <h1 class="title is-1"> Past Sponsors</h1>
...
+                  <a href="#"><i class="fab fa-linkedin icon "></i></a>
```
Comment (championshuttler) [inline index.html]: What's your review about linkedin profile @suhailsinghbains ?

**Comment by championshuttler** [comment]
Hi @akshat0047 thanks for your review it almost looks good , i added few comments for changing. Rest LGTM :+1: You can add your profile as well :)

File: index.html (hunk 1 above)
Comment (suhailsinghbains) [reply index.html]: Yes, removing Instagram & Pinterest icons will be a good idea!
And also, adding Mozilla Icon will be a nice touch for Mozillian Id

File: index.html (hunk 3 above)
Comment (suhailsinghbains) [reply index.html]: As I don't have an account on Facebook,
I should be having LinkedIn Icon and you both can have Facebook icons, so, that, number of links for each team member is equal. :)

**Comment by suhailsinghbains** [comment]
This is for a small-sized display, the images are getting out their respective position, please check image shared.
image
Note: for xs display, it is looking perfect.

File: index.html (hunk 3 above)
Comment (akshat0047) [reply index.html]: @suhailsinghbains linkedin icon is already present so it'll not be replaced with the facebook one, should I remove the fb-icon from your card or add some other link. (mozilla and telegram icons are added in place of instagram and pinterest respectively.

**Comment by akshat0047** [comment]
@championshuttler  thank you, l'll add my profile too in the update of changes suggested.

File: index.html (hunk 3 above)
Comment (suhailsinghbains) [reply index.html]: @akshat0047 
Yeah, remove fb-icon from my card and remove linkedin icon from other people's card. 
Hope I make sense, feel free to revert :)

**Comment by akshat0047** [comment]
I've updated the pr and now need the links and bio of all members, also, the name member with lastname "bansal".

**Comment by suhailsinghbains** [comment]
Hi @akshat0047 [Here](https://github.com/Moz-NIEC/moz-hack-3/issues/33#issuecomment-422813941), you can find what you are looking for.
Whatever I have not mentioned, just let me know and I'll try to provide :)

**Comment by akshat0047** [comment]
Updated the links provided

**Comment by suhailsinghbains** [comment]
Looking great.
Thanks for solving the bug @akshat0047

**Pull request merged**